### PR TITLE
new package: py-pydotplus

### PR DESCRIPTION
**Pull request by Sinan81**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pydotplus/package.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    depends_on('py-tox', type='test')
+    depends_on('py-flake8', type='test')
+    depends_on('py-pytest', type='test')
+    depends_on('py-pytest-cov', type='test')
+    depends_on('py-coverage', type='test')
+    depends_on('py-coveralls', type='test')
+    depends_on('py-sphinx', type='build', when='+docs')
+    depends_on('py-sphinx-rtd-theme', type='build', when='+docs')
+
+    conflicts('python@2.8:3.3.99')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pydotplus/package.py]:
```suggestion
    depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-pydotplus/package.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    version('2.0.2', sha256='91e85e9ee9b85d2391ead7d635e3d9c7f5f44fd60a60e59b13e2403fa66505c4')
+
+    variant('docs', default=False, description='Build docs')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-pyparsing@2.0.1:', type=('build', 'run'))
+
+    depends_on('py-tox', type='test')
+    depends_on('py-flake8', type='test')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pydotplus/package.py]: Tox and flake8 are probably only required for Travis, not to run the unit tests

File: var/spack/repos/builtin/packages/py-pydotplus/package.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    depends_on('py-pytest', type='test')
+    depends_on('py-pytest-cov', type='test')
+    depends_on('py-coverage', type='test')
+    depends_on('py-coveralls', type='test')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pydotplus/package.py]: These last 3 are probably only required for Travis, not to run the unit tests

File: var/spack/repos/builtin/packages/py-pydotplus/package.py (hunk 2 above)
Comment (Sinan81) [reply var/spack/repos/builtin/packages/py-pydotplus/package.py]: Yeah, I wasn't sure. Should I remove them?

File: var/spack/repos/builtin/packages/py-pydotplus/package.py (hunk 3 above)
Comment (Sinan81) [reply var/spack/repos/builtin/packages/py-pydotplus/package.py]: Should I remove them?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pydotplus/package.py]: Remove them and try building with `--test=root`. If it works, they aren't needed. If it doesn't, they are.

Comment (Sinan81) [reply var/spack/repos/builtin/packages/py-pydotplus/package.py]: it worked. will be removing them shortly.

**Pull request merged**